examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Also worth mentioning, there are different kinds of interfaces possible, for e.g.
```js
<Jss>
{styles}
</Jss>
```

This way it can be used without a HOC, I have an is issue for that.